server/server.go

```diff
@@ -180,7 +176,7 @@ func (m *Command) SetupServer() error {
 		if err != nil {
 			gossipHost = m.Config.Host
 		}
-		gossipNodeSet := gossip.NewGossipNodeSet(m.Config.Host, gossipHost, gossipPort, gossipSeed, m.Server)
+		gossipNodeSet := gossip.NewGossipNodeSet(m.Config.Host, gossipHost, gossipPort, gossipSeed, m.Server, m.Server.LogOutput)
```

Comment:
can NewGossipNodeSet rely on m.Server to get the LogOutput, or do we think that's a bad idea?